src/gobstuf/stuf/brp/response_mapping.py

```diff
@@ -57,6 +57,13 @@ def filter_none_values(obj):
                     value = filter_none_values(v)
                     if value:
                         result[k] = value
+                elif isinstance(v, list):
+                    result_list = []
+                    for obj in v:
+                        value = filter_none_values(obj)
+                        if value:
+                            result_list.append(value)
+                    result[k] = result_list
```

Comment:
Maybe replace this with a list comprehension? Seems a bit easier to understand. Also gets rid of the noqa: C901 

`result[k] = [item for item in [ filter_none_values(obj) for obj in v ] if item]`